protocols/ipfs.ts

```diff
@@ -153,18 +176,22 @@ export class IPFSProtocol implements Protocol<Static<typeof IPFSProtocolFields>>
           '/p2p-circuit'
         ],
         announce: [
-          `/ip4/${publicIP}/tcp/${tcpPort}`,
-          `/ip4/${publicIP}/tcp/${wsPort}/ws`
+          ...(publicIP !== '0.0.0.0' ? [`/ip4/${publicIP}/tcp/${tcpPort}`] : []),
+          ...(publicIP !== '0.0.0.0' ? [`/ip4/${publicIP}/tcp/${wsPort}/ws`] : []),
+          ...(this.options.useWebRTC === true && webrtcPort !== null && publicIP !== '0.0.0.0'
+            ? [`/ip4/${publicIP}/udp/${String(webrtcPort)}/webrtc-direct`]
+            : []),
+          '/dnsaddr/bootstrap.libp2p.io/p2p/QmNnooDu7bfjPFoTZYxMNLWUQJyrVwtbZg5gBMjTezGAJN'
```

Comment:
Why are you announcing the public bootstrapper?

Reply:
Yeah, I'll remove it. That was just for temporary testing to announce addresses to bypass NAT.